api/test/handlers/scrapePosts/scrapePosts.spec.js

```diff
@@ -0,0 +1,67 @@
+const {expect} = require('chai');
+const scrapePosts = require('../../../src/handlers/scrapePosts/scrapePosts.js');
+
+describe('scrapePosts', () => {
+  let dbSpy;
+  let feedManagerSpy;
+  let authorsStub;
+  let feedsStub;
+  let postsStub;
+
+  beforeEach(() => {
+    authorsStub = {};
+    feedsStub = {};
+    postsStub = {};
+
+    dbSpy = {
+      getAuthorsCalled: false,
+      getAuthors: function() {
```

Comment:
Does that always work? I thought there is a while thing about binding `this` and arrow functions

Reply:
sadly it's a problem with `this`, if we use arrow functions then it thinks `this` is the context of the `getAuthors` function rather than the `dbSpy`: lol js 🌀